app/gen-server/lib/homedb/UsersManager.ts

```diff
@@ -432,6 +433,11 @@ export class UsersManager {
         user.options = {...(user.options ?? {}), authSubject: userOptions.authSubject};
         needUpdate = true;
       }
+      const today = moment().startOf('day');
+      if (!user.lastConnectionAt || !today.isSame(moment(user.lastConnectionAt).startOf('day'))) {
+        user.lastConnectionAt = today.toDate();
```

Comment:
In line 403 we have this line:
```
nowish.setMilliseconds(0);
```
Do you think we need it here?

Reply:
No, we does not need it because we take the `moment()` starting on the day

Reply:
Ok, one last question about the `date` column. I'm a little bit concerned here, what is actually stored and saved when we have 2 home servers in two different timezones. I did a bit of debugging and in Sqlite it stored:
`2024-06-20 22:00:00.000` (so UTC time of `day start` in my timezone +2), but in postgress only `2024-06-21`. But in code both those values are mapped to `Date` type, and I think represented in local time.

Reply:
I'm thinking, if it would be simpler if we just use `datetime` column in both Sqlite and Postgres. And treat it as `unix timestamp`, the same way as we do for any other `created_at` or `updated_at` columns. It will be easier to compare those if needed, and it is simpler to reason about, as having unix timestamps in database is a common approach.

Math here is also very simple:
```
const timeStamp = Math.floor(Date.now() / 1000); // unix timestamp seconds from epoc
const startOfDay = timestamp - (timestamp % 86400 /*24h*/); // start of a day in seconds since epoc
```

Reply:
Oh yes, I hadn't seen that, thanks. In fact I think it's a good thing to use datetime, I will change it